.github/workflows/build.yml

```diff
@@ -30,6 +34,9 @@ jobs:
         echo "CABAL_BUILDDIR=$CABAL_BUILDDIR"
         echo "CABAL_BUILDDIR=$CABAL_BUILDDIR" >> $GITHUB_ENV
 
+    - name: Set cache version
+      run: echo "CACHE_VERSION=9w76Z3Q"          >> $GITHUB_ENV
```

Comment:
Where does `9w76Z3Q` come from?

Reply:
Random generated string.  It's used as a way to "invalidate the cache".  For example if I want to know how long a full build takes, I change the string.